src/utils/gh-auth.js

```diff
@@ -109,12 +109,12 @@ const authWithToken = async () => {
     const { login: user } = await octokit.users.getAuthenticated()
     return { token, user, provider: 'github' }
   }
-  const error = new Error('Github authentication failed')
+  const error = new Error('GitHub authentication failed')
```

Comment:
since I noticed it, should I change this to:

```js
throw new Error('GitHub authentication failed')
```

and get rid of the variable?

Reply:
Yep, that works 👍

Reply:
I think I will make a separate PR later if you agree since this is a small refactoring:

```diff
 src/utils/gh-auth.js | 22 +++++++++-------------
 1 file changed, 9 insertions(+), 13 deletions(-)

diff --git a/src/utils/gh-auth.js b/src/utils/gh-auth.js
index fca6a3ea..b9f1e1ba 100644
--- a/src/utils/gh-auth.js
+++ b/src/utils/gh-auth.js
@@ -102,30 +102,26 @@ const getPersonalAccessToken = async () => {
  */
 const authWithToken = async () => {
   const { token } = await getPersonalAccessToken()
-  if (token) {
-    const octokit = new Octokit({
-      auth: `token ${token}`,
-    })
-    const { login: user } = await octokit.users.getAuthenticated()
-    return { token, user, provider: 'github' }
+  if (!token) {
+    throw new Error('GitHub authentication failed')
   }
-  const error = new Error('Github authentication failed')
-  throw error
+
+  const octokit = new Octokit({ auth: `token ${token}` })
+  const { login: user } = await octokit.users.getAuthenticated()
+
+  return { token, user, provider: 'github' }
 }
 
 /**
- * Get a github token
+ * Get a GitHub token
  * @returns {Promise<Token>} Returns a Promise with a token object
  */
 const getGitHubToken = async () => {
   log('')
 
   const withNetlify = await promptForAuthMethod()
-  if (withNetlify) {
-    return await authWithNetlify()
-  }
 
-  return await authWithToken()
+  return withNetlify ? await authWithNetlify() : await authWithToken()
 }
 
 module.exports = { getGitHubToken, authWithNetlify }

```